contracts/ExitHandler.sol

```diff
@@ -123,6 +123,42 @@ contract ExitHandler is DepositHandler {
     );
   }
 
+  function startDepositExit(uint256 _depositId) public payable {
+    require(msg.value >= exitStake, "Not enough ether sent to pay for exit stake");
+    // check that deposit exits
+    Deposit deposit = deposits[uint32(_depositId)];
+    require(deposit.owner == msg.sender, "Only deposit owner can start exit");
+    require(deposit.amount > 0, "deposit has no value");
+    require(exits[bytes32(_depositId)].amount == 0, "The exit of deposit has already been started");
+    require(!exits[bytes32(_depositId)].finalized, "The exit for deposit has already been finalized");
+    
+    uint256 priority;
+    if (isNft(deposit.color)) {
+      priority = (nftExitCounter << 128) | uint128(_depositId);
+      nftExitCounter++;
+    } else {      
+      priority = getERC20ExitPriority(uint32(now), bytes32(_depositId), 0);
```

Comment:
I think using _now_ here is iffy. The value here should be the timestamp of the period that was latest when the deposit was made. Using _now_, valid deposit exits will have lower priority than invalid out of nowhere exits included in a period before _now_. This was exactly the reason we concluded it makes the most sense to have periods created on deposits, but I forgot about it when we last discussed. If we want to keep this exit scheme, we have to associate with every deposit the current latest period.

Reply:
replaced with time of deposit